src/entities/session/model/types.ts

```diff
@@ -0,0 +1,8 @@
+export type User = {
+  email: string;
+  name: string;
+  nickname: string;
+  picture: string;
+  sub: string;
+  updated_at: string;
```

Comment:
we use CamelCase

Reply:
no problem) but the thisn is that the server makes a response with such data, if I`ll change it to camalCase so then it would be to totally different constants. Server will response with { name: ..., nickname: ..., .... } and i will expect nickName)

Reply:
we cannot use the types provided by an external library. We do everything the way we need it. You can write the constants you need in the api layer. And return them. And if you need to post something on the server, it's the same, only the other way around.

Reply:
can you provide an example? because I totally lost the point) 
so I`m recieving a response from the authAPI, the response is object with data provided by auth server. So how can i change it and mark needed types?